src/lib/blkalloc/blk_allocator.h

```diff
@@ -65,37 +63,28 @@ struct BlkAllocConfig {
     const uint32_t m_align_size;
     const blk_num_t m_capacity;
     const blk_num_t m_blks_per_portion;
+    const bool m_persistent{false};
     const std::string m_unique_name;
-    bool m_auto_recovery{false};
-    bool m_realtime_bm_on{false}; // only specifically turn off in BlkAlloc Test;
 
 public:
-    BlkAllocConfig(uint32_t blk_size, uint32_t align_size, uint64_t size, const std::string& name = "",
-                   bool realtime_bm_on = true) :
+    BlkAllocConfig(uint32_t blk_size, uint32_t align_size, uint64_t size, bool persistent,
+                   const std::string& name = "") :
```

Comment:
Removed realtime_bm_on, since everything is folded inside VDev. Will introduce similar functionality for debugging on subsequent PRs.